src/callbacks_step/analysis_dg2d_parallel.jl

```diff
@@ -139,14 +139,13 @@ end
 function integrate_via_indices(func::Func, u,
                                mesh::ParallelTreeMesh{2}, equations, dg::DGSEM, cache,
```

Comment:
We also got rid of `invoke` here, so we need to dispatch on `u::TrixiMPIArray` instead (only for this method)

Reply:
👍